srcpkgs/highway/template

```diff
@@ -1,23 +1,33 @@
 # Template file for 'highway'
 pkgname=highway
-version=1.1.0
+version=1.2.0
 revision=1
 build_style=cmake
-configure_args="-DHWY_SYSTEM_GTEST=ON -DHWY_ENABLE_EXAMPLES=OFF"
+configure_args="-DHWY_SYSTEM_GTEST=ON -DHWY_ENABLE_EXAMPLES=OFF -DBUILD_SHARED_LIBS=ON"
```

Comment:
Previously static libs were being used, I think generally void prefers to use shared libs, but I can revert back to static ones if preferred.